project-base/storefront/components/Pages/Styleguide/StyleguideColors.tsx

```diff
@@ -0,0 +1,53 @@
+import { StyleguideSection } from './StyleguideElements';
+import React from 'react';
+import { twJoin } from 'tailwind-merge';
+
+class ColorItem {
+    constructor(
+        public name: string,
+        public className: string,
+        // eslint-disable-next-line no-empty-function
+    ) {}
+}
```

Comment:
Just a question, why to use classes instead of just regular objects?

Reply:
Because it easier to create new values